### PR TITLE
[CMS-301] Update IC docs to point to new composer-based WP upstream

### PR DESCRIPTION
**Pull request by jspellman814**
## Summary

This change will instruct customers to use our new upstream, wordpress-composer-managed, for WordPress IC sites.

## Remaining Work and Prerequisites

The following changes still need to be completed:

- [x] [wordpress-composer-managed repo](https://github.com/pantheon-systems/wordpress-composer-managed/) is cloned to [pantheon-upstreams organization ](https://github.com/pantheon-upstreams/)
- [ ] wordpress-composer-managed reaches LA/GA release phase

### Dependencies and Timing

- [ ] Remaining work above needs to be completed before this is released

**Release**:
- [ ] After the dependencies above are complete

--------------------------------------------------

## Post Launch

**Do not remove** - To be completed by the docs team upon merge:

- [ ] Redirect `/docs/old-path/` => `/docs/new-path/` (if applicable)
- [ ] Include/exclude pages ^ respectively within docs search service provider (if applicable)
- [ ] For Heroes - add a props post to the [discussion board](https://discuss.pantheon.io/c/pantheon-platform/documentation/17).
- [ ] Remove from the [project board](https://github.com/pantheon-systems/documentation/projects/14)

### CODE REVIEW

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | [Live](https://pantheon.io/docs) | [pr-7488](https://pr-7488--pantheon-docs.my.pantheonfrontend.website/docs) | Difference |
| --- | --- | --- | --- |
| Performance Score | `43` | `48` | `5` |
| Best Practice Score | `86` | `86` | `0` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `98` | `98` | `0` |

Lighthouse Audit Reports:

- [`pr-7488` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/39396/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/39396/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)


[Preview Environment](https://pr-7488--pantheon-docs.my.pantheonfrontend.website/docs)

File: source/content/guides/integrated-composer.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ Integrated Composer is a Pantheon platform feature that extends Composer <Popove
 
 ### WordPress with Integrated Composer
 
-1. [Fork the Pantheon-maintained repository](/create-custom-upstream#create-and-host-the-repository-remotely) from [https://github.com/pantheon-upstreams/wordpress-project](https://github.com/pantheon-upstreams/wordpress-project).
+1. [Fork the Pantheon-maintained repository](/create-custom-upstream#create-and-host-the-repository-remotely) from [https://github.com/pantheon-upstreams/wordpress-composer-managed](https://github.com/pantheon-upstreams/wordpress-composer-managed).
```
Comment (scottbuscemi) [inline source/content/guides/integrated-composer.md:26]: @jspellman814 Forking not needed

File: source/content/guides/wordpress-composer/06-wordpress-ic.md [hunk 2]
```diff
@@ -29,7 +29,7 @@ Pantheon upstreams influence the Composer packages that are included in downstre
 
 Follow the steps in this section to create a new WordPress site using Integrated Composer and Upstream.
 
-1. Fork the [Pantheon-maintained WordPress Upstream repository](https://github.com/pantheon-upstreams/wordpress-project).
+1. Fork the [Pantheon-maintained WordPress Upstream repository](https://github.com/pantheon-upstreams/wordpress-composer-managed).
```
Comment (scottbuscemi) [inline source/content/guides/wordpress-composer/06-wordpress-ic.md:32]: @jspellman814 Forking unnecessary

File: source/content/guides/integrated-composer.md [hunk 3]
```diff
@@ -335,9 +324,13 @@ code/web/wp-content/
 └─ plugins/
 └─ themes/
 ```
+<<<<<<< HEAD
+See the `.gitignore` file for WordPress [here](https://github.com/pantheon-upstreams/wordpress-composer-managed/blob/master/.gitignore).
```
Comment (kporras07) [inline source/content/guides/integrated-composer.md:328]: Merge conflicts?

**Comment by wordsmither** [comment]
@scottbuscemi , @kporras07 , so this one hasn't been touched in about a year.  Is it still something that requires work?

**Comment by scottbuscemi** [comment]
@wordsmither The work on the new bedrock composer upstream has been on hold but will be ramping up again soon. Would love to keep this open if possible :)

**Comment by stevector** [comment]
> The work on the new bedrock composer upstream has been on hold but will be ramping up again soon.

I hear that work is now progressing. @scottbuscemi would you prefer to update this PR? Given how old it is, I wouldn't be surprised if it's easier to close this PR and just make a new one when the time comes.

**Comment by stevector** [comment]
@rachelwhitton it looks like https://github.com/pantheon-systems/documentation/pull/9031 will replace this PR. Can you confirm?

**Pull request closed without merging**